Source/Scene/Scene.js

```diff
@@ -955,6 +997,97 @@ define([
         return undefined;
     }
 
+    // These values were found by sampling the density at certain views and finding at what point culled tiles impacted the view at the horizon.
```

Comment:
Actually, it would be nice to put all this code in the `Fog` class as `@private` members (just used by `Scene`) to keep all the Scene code at a similar level of abstraction.